composables/masto/translate.ts

```diff
@@ -63,9 +97,14 @@ export function useTranslation(status: mastodon.v1.Status | mastodon.v1.StatusEd
     translations.set(status, reactive({ visible: false, text: '', success: false, error: '' }))
 
   const translation = translations.get(status)!
+  const userSettings = useUserSettings()
+
+  const shouldTranslate = 'language' in status && status.language && status.language !== to
```

Comment:
Maybe we could add a check here, if the `to` language is supported for translation. Otherwise, the translation will not work any way. See line 11.

Reply:
Yes, let's do this 👍🏼